README.md

```diff
@@ -148,8 +148,8 @@ We carefully implemented sbt-crossproject to be mostly source compatible with Sc
 In `project/plugins.sbt`:
 
 ```scala
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.23")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.3.4")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.0.16")
```

Comment:
This one must stay 0.6.x, since the section is about being compatible with 0.6.x' `crossProject`.
```suggestion
addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.33")
```